src/CrazyradioThread.cpp

```diff
@@ -227,13 +227,23 @@ void CrazyradioThread::run()
                 const std::lock_guard<std::mutex> lock(con->queue_send_mutex_);
                 if (!con->queue_send_.empty())
                 {
-                    const auto p = con->queue_send_.top();
-                    ack = radio.sendPacket(p.raw(), p.size());
-                    ++con->statistics_.sent_count;
-                    if (ack)
+                    auto p = con->queue_send_.top();
+                    if (con->broadcast_)
                     {
+                        p.setSafelink(0);
```

Comment:
this should not be needed. The safelink bits are simply ignored if safelink is disabled.

Reply:
Yep, you're right. The reason I needed to do this previously was because I was testing with a `Connection` with safelink enabled.